ddl/index.go

```diff
@@ -678,7 +684,19 @@ func (w *addIndexWorker) handleBackfillTask(d *ddlCtx, task *reorgIndexTask) *ad
 		}
 
 		handleRange.startHandle = nextHandle
-		if handleRange.startHandle >= handleRange.endHandle {
+		finishTask := false
+		// if nextHandle is math.Int64, we can only use handleOutOfRange to indicates the nexHandle is handled.
+		if handleOutOfRange && nextHandle == handleRange.endHandle {
```

Comment:
Is `handleOutOfRange` implies `nextHandle == handleRange.endHandle`?

Reply:
no. handleOutOfRange just mean current batch is handled out of range.